pallets/ocex/src/lib.rs

```diff
@@ -159,19 +163,29 @@ pub mod pallet {
 
 	#[pallet::error]
 	pub enum Error<T> {
+		///RegisterationShouldBeSignedByMainAccount
```

Comment:
would love to see more descriptive message here - something similar to old errors below with proper description would be nice